app/workers/form5655/vba_submission_job.rb

```diff
@@ -5,13 +5,26 @@
 module Form5655
   class VBASubmissionJob
     include Sidekiq::Worker
+    include SentryLogging
 
     sidekiq_options retry: false
 
+    sidekiq_retries_exhausted do |job, _ex|
+      user_uuid = job['args'][1]
+      UserProfileAttributes.find(user_uuid).destroy
+    rescue => e
+      log_message_to_sentry(
+        "Failed to delete user attributes #{user_uuid}",
```

Comment:
You could probably just destroy if `UserProfileAttributes.find(user_uuid)` exists and not worry about rescuing. If it doesn't exist then it's no big deal that the destroy failed (since it'll be gone anyways)

Reply:
Just throwing in `&` is prob sufficient: `UserProfileAttributes.find(user_uuid)&.destroy`